src/js/defaultTheme/styles.scss

```diff
@@ -45,6 +45,10 @@
 
 }
 
+.mstr-hide-controls {
+    animation: hide-bars-animation 0.4s forwards;
```

Comment:
:3c

Haha well I sorta did to get the animation working but I am just pointing at a better way to transition it. It's quite unlikely that this will cause the UI to break it's neater (and easier to maintain) to use `transition`'s